src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -19,23 +19,24 @@ import HeaderLayout from './HeaderLayout';
 import HubLinks from './HubLinks';
 import NewsletterSignup from './NewsletterSignup';
 import HelpdeskInfo from './HelpdeskInfo';
+import MhvRegistrationAlert from './MhvRegistrationAlert';
 import {
   isLOA3,
   isVAPatient,
   personalizationEnabled,
   helpdeskInfoEnabled,
+  hasMhvAccount,
 } from '../selectors';
 import UnregisteredAlert from './UnregisteredAlert';
 
 const LandingPage = ({ data = {}, recordEvent = recordEventFn }) => {
   const { cards = [], hubs = [] } = data;
   const verified = useSelector(isLOA3);
-  const registered = useSelector(isVAPatient);
+  const registered = useSelector(isVAPatient) && verified;
```

Comment:
This is a guard against having an unverified patient showing as registered which is the rule for this flag. We have been incorrectly setting these flags in some of our tests and this will guard against any data issues for that flag. It does not change its meaning.

Reply:
Agreed, although maybe calling it registered works here since being verified is part of that flag.